tests/ui-toml/absolute_symbol_paths/absolute_symbol_paths.rs

```diff
@@ -1,4 +1,7 @@
 //@aux-build:proc_macros.rs:proc-macro
```

Comment:
```suggestion
//@aux-build:../../ui/auxiliary/proc_macros.rs:proc-macro
```

No need to copy `proc_macros.rs`

Reply:
I guess that works with `ui_test`, before it wouldn't on linux. neat